src/api/journalEntries/journalEntries.ts

```diff
@@ -30,7 +30,7 @@ export const useJournalEntries = (authorId: string | undefined) => {
     });
 };
 
-const authorJournalEntries = gql`
+const authorJournalEntry = gql`
```

Comment:
Since it is only getting one entry for one author, the naming should be singular